airflow/providers/docker/operators/docker.py

```diff
@@ -209,6 +212,7 @@ def __init__(
         log_opts_max_size: str | None = None,
         log_opts_max_file: str | None = None,
         ipc_mode: str | None = None,
+        skip_exit_code: int = 99,
```

Comment:
Just a question. Should we actually provide any default value for that? Or is it user responsibility?
In theory some current users pipelines might return 99 status code

Reply:
I agree, I just used the same default value used in the `BashOperator`

Reply:
I don't think it make sense to copy default value from `BashOperator`.

Reply:
And also we need to specify annotation as `int | None` it is not raise any static checks because we have default value 99 however user might provide `None` and this is expected value in `execute`